### PR TITLE
Tutorial March 7 Daily PR

### PR DESCRIPTION
**Pull request by pe7r**
Practicing with class components, components' state and events.

### CODE REVIEW

File: src/components/ToDoItem.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import './ToDoItem.css'
+
+function ToDoItem(props) {
```
Comment (ruslan-palahin) [inline src/components/ToDoItem.js]: 1. Please use arrow functions when it's possible. Example:
const ToDoItem = () => {
...
}; 

2. Please use ES6 destruction for props. Example:
const ToDoItem = ({text}) => ...

3. Please read about explicit return. Example:
```
const ToDoItem = ({text}) => (
  <div className="todo-item">
			<input type="checkbox" />
			<h3>{props.text}</h3>
		</div>
);
```
4. In a future PropTypes will be also a must (next week)

File: src/components/ToDoList.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import ToDoItem from './ToDoItem.js'
+import './ToDoList.css'
+import items from '../items.js'
+
+class ToDoList extends React.Component {
+	constructor() {
```
Comment (ruslan-palahin) [inline src/components/ToDoList.js:7]: 1. Do not forget about passing constructor arguments to super (parent) class. Example:
constructor(props) {
super(props)
		this.state = {
			todos: items
		}
	}

**Review by ruslan-palahin — CHANGES_REQUESTED** [review]

File: src/components/ToDoList.js (hunk 2 above)
Comment (pe7r) [reply src/components/ToDoList.js:7]: Do I need to do a constructor in all of my class components?

File: src/components/ToDoItem.js (hunk 1 above)
Comment (pe7r) [reply src/components/ToDoItem.js]: I fixed tabulation and redo funcs into arrow funcs. Will learn about destruction for props and explicit return

File: src/components/ToDoList.js (hunk 2 above)
Comment (ruslan-palahin) [reply src/components/ToDoList.js:7]: No, define it only if you have some logic for constructor. But if you define it - do not forget about "super(props)"

**Review by ruslan-palahin — CHANGES_REQUESTED** [review]

**Pull request merged**